weave/trace/weave_client.py

```diff
@@ -1181,11 +1177,11 @@ def _add_score(
 
         freq = FeedbackCreateReq(
             project_id=self._project_id(),
-            weave_ref=call_ref_uri,
-            feedback_type="wandb.runnable." + score_name,
+            weave_ref=weave_ref_uri,
+            feedback_type="wandb.runnable." + runnable_ref.name,
```

Comment:
may want to pull out this `wandb.runnable.` magic val as a const

Reply:
Done